tests/RedshiftDatatypeTest.php

```diff
@@ -24,113 +24,33 @@ public function testInvalidType()
         }
     }
 
-    public function testValidNumericLengths()
-    {
-        new Redshift("numeric");
-        new Redshift("NUMERIC");
-        new Redshift("NUMERIC", ["length" => ""]);
-        new Redshift("INT", ["length" => ""]);
-        new Redshift("NUMERIC", ["length" => "37,0"]);
-        new Redshift("NUMERIC", ["length" => "37,37"]);
-        new Redshift("NUMERIC", ["length" => "37"]);
-        new Redshift('NUMERIC', [
-            'length' => [
-                'numeric_precision' => '37',
-                'numeric_scale' => '0'
-            ]
-        ]);
-        new Redshift('NUMERIC', [
-            'length' => [
-                'numeric_precision' => '37',
-                'numeric_scale' => '37'
-            ]
-        ]);
-        new Redshift('NUMERIC', [
-            'length' => [
-                'numeric_precision' => '37'
-            ]
-        ]);
-        new Redshift('NUMERIC', [
-            'length' => [
-                'numeric_scale' => '37'
-            ]
-        ]);
-    }
-
     /**
-     * @dataProvider invalidNumericLengths
-     * @param $length
+     * @param string $columnType
+     * @param array $lengthOption
+     * @param string $expectedOutput
+     * @throws InvalidOptionException
+     * @dataProvider validLengthsProvider
      */
-    public function testInvalidNumericLengths($length)
+    public function testValidLengths($columnType, $option, $expectedOutput)
```

Comment:
čekal bych prostě valid options a invalid options - ne nutně extra test jen na délku. Když se bude řešit nějaká logika na nullable nebo na default, tak bude extra test? Imho to není potřeba. A klidně by se mohly validovat všechny hodnoty, co datatype poskytuje...

Reply:
Z kódu to vypadá, že "length", "nullable", "default", "compression" jsou jediné možné

Reply:
Kdyby to bylo tak, tak nejsou potřeba ty první dva testy v téhle třídě...